packages/react-native-editor/ios/Podfile

```diff
@@ -5,6 +5,10 @@ platform :ios, '13.0'
 require_relative '../../../node_modules/react-native/scripts/react_native_pods'
 require_relative '../../../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+# Changes Aztec reference to last commit from https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1346 PR.
+# ONLY FOR TESTING, IT SHOULD BE DELETED ONCE THE AZTEC PR IS MERGED!
+pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => 'd68c76ec8f01490c741077dccefc17b17ce19f2d'
+
```

Comment:
I updated the Aztec-iOS reference so it points to the PR (https://github.com/wordpress-mobile/AztecEditor-iOS/pull/134) where the `shouldRecalculateTypingAttributesOnDeleteBackward` variable was added.